strings.lua

```diff
@@ -729,16 +729,16 @@ Aufzüge, Treppen und unterirdische Eingänge finden Sie in den Assets, suchen S
 			["WallConcreteModernDesc"] = "Betonwand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
 			["WallConcretePlainName"] = "Betonwand, schlicht",
 			["WallConcretePlainDesc"] = "Betonwand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallFenceMattoniName"] = "Backsteinzaun",
-			["WallFenceMattoniDesc"] = "Backsteinzaun, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
+			["WallFenceMattoniName"] = "Backsteinmauer",
+			["WallFenceMattoniDesc"] = "Backsteinmauer, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
 			["WallIronName"] = "Eisenzaun",
 			["WallIronDesc"] = "Eisenzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
 			["WallIronGlassCopperName"] = "Wand aus Eisen und Glas",
 			["WallIronGlassCopperDesc"] = "Eisen- und Glaswand, auch für Tunnels geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallStaccionataFsName"] = "1,4m Betonzaun",
-			["WallStaccionataFsDesc"] = "1,4m Betonzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
-			["WallStaccionataFsTallName"] = "2m Betonzaun",
-			["WallStaccionataFsTallDesc"] = "2m Betonzaun, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
+			["WallStaccionataFsName"] = "1,4m Betonwand",
+			["WallStaccionataFsDesc"] = "1,4m Betonwand, auch für Tunnel geeignet. Stützwand mit <m> und <n> einstellen.",
+			["WallStaccionataFsTallName"] = "2m Betonwand",
```

Comment:
Some of these things are walls, others are fences, so don't swap them.

Reply:
Allright.
The thing is, the german word "Zaun" is not meant to be used for something made of bricks or concrete.
If it's at the the side of a tunnel, it's a wall ("Wand"), if it's in the middle as a divider, it's a wall ("Mauer" or even better: "Trennwand" which literally means "divider wall").